modules/ROOT/partials/configuration/uploadcare_srcset_steps.adoc

```diff
@@ -0,0 +1,23 @@
+[[uploadcare-srcset-steps]]
+== `uploadcare_srcset_steps`
+
+Defines an array of numeric values representing the widths for responsive images. These widths are used to generate the `srcset` attribute for images, allowing browsers to select the most appropriate resolution based on the device's display and network conditions.
+
+Customize this array to include only the widths you need for your application. For example, you can reduce the number of resolutions for faster processing or add specific sizes required for your layout.
```

Comment:
I wonder if we should include an example here. Basically if you upload an image that is 780 pixels wide it will generate a `srcset` with with downscaled images for each of the sizes that are below that point so in this case `100, 200, 300, 500, 750` so if the browser viewport is narrow it will pick on the smaller images for example on a mobile device.

Reply:
Updated this page content, please re-review the updated documentation and example added.